lib/resque/tasks.rb

```diff
@@ -19,6 +19,8 @@
         worker.very_verbose = ENV['VVERBOSE']
       end
       worker.term_timeout = ENV['RESQUE_TERM_TIMEOUT'] || 4.0
+      worker.pre_shutdown_timeout = ENV['RESQUE_PRE_SHUTDOWN_TIMEOUT'] || 0.0
```

Comment:
Cool, seems like the tests have errored out for this build.  I'm going to rebase and run them again.